feat(rn): release notes v1.59.2 (#643)
replicatedhq/kots.io: content/release-notes/1.59.2.md

---
date: 2022-01-18T10:00:00-05:00
linktitle: "1.59.2"
title: "1.59.2"
kubernetes: "1.19, 1.20, and 1.21"
---

{{<fixes>}}
* Adds a more descriptive error message to the KOTS CLI when the provided host path does not exist for snapshots storage.
* Fixes a bug that caused the `Send bundle to vendor` link to be shown when this feature is not enabled.
* Cleanup to CSS and resolving style issues.

> * Fixes a bug where excluded helm charts can change between 'UseHelmInstall: true' and 'UseHelmInstall: false' without errors, bypassing the normal check.

Reviewer: ```suggestion
* Fixes a bug where excluded helm charts could not change between 'UseHelmInstall: true' and 'UseHelmInstall: false' without errors.
```
